src/cpp/library/cstdlib

```diff
@@ -154,14 +154,15 @@ char get_char(int digit) {
 	return charstr[digit];
 }
 
-void rev(char *p) {
-	char *q = &p[strlen(p) - 1];
+void rev(char *p) {	
+	char *q = p + strlen(p) - 1;	
```

Comment:
Is this needed? ESBMC doesn't report error or warning on the old code.

Reply:
Yes, this is not necessary, I checked it and it doesn't seem to give an error in the latest version anymore.